src/main/java/seedu/address/logic/commands/AddTransactionCommand.java

```diff
@@ -67,6 +67,10 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         List<Person> lastShownList = model.getFilteredPersonList();
+        if (lastShownList.isEmpty()) {
+            throw new CommandException(String.format(Messages.MESSAGE_EMPTY_PERSON_LIST, COMMAND_WORD));
+        }
+
```

Comment:
could you change the other commands to use this check too? I think its good

Reply:
When I add this into other commands it causes some tests in LogicManagerTest to fail, so I think maybe it's better if whoever is responsible for the specific command to update? In case anything is changed by mistake